src/Apps/CollectorProfile/Routes/Saves2/useCheckIfArtworkListsEnabled.ts

```diff
@@ -6,6 +6,7 @@ export const useCheckIfArtworkListsEnabled = () => {
   const isFeatureFlagEnabled = useFeatureFlag("force-enable-artworks-list")
   const { user } = useSystemContext()
   const isArtsyEmployee = isArtsyEmail(user?.email ?? "")
+  const isIntegrityUser = user?.email === "cypress+test@example.com"
```

Comment:
**suggestion**: make it a separate function. Something like this

```javascript
const isIntegrityEmail = (email: string) => {
  return email === "cypress+test@example.com"
}

...
// src/Apps/CollectorProfile/Routes/Saves2/useCheckIfArtworkListsEnabled.ts
const isIntegrityUser = isIntegrityEmail(user?.email ?? "")

// src/Apps/CollectorProfile/collectorProfileRoutes.tsx
const isIntegrityUser = isIntegrityEmail(context?.user?.email ?? "")
```

Reply:
I was keeping it simple, since this will be removed in a matter of days anyway. 

Will leave as is if you don't mind. (Unless we think it's good to have this as a helper in the long term 🤔)